docs/content/resources/package.apt.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "package.apt"
+slug: "package-apt"
+date: "2016-11-01"
+menu:
+  main:
+    parent: resources
+---
+
+
+apt Package manages system packages with `apt` and `dpkg`. It assumes that
+both `apt` and `dpkg` are installed on the system, and that the user has
+permissions to install, remove, and query packages.
+
```

Comment:
should we add a note about using `group` to ensure that multiple apt package installations do not run in parallel? (at least until we add an implicit group into the apt resource)